africanus/calibration/tests/test_utils.py

```diff
@@ -97,7 +110,64 @@ def make_dual_pol_data(sigma_n, n_dir, sigma_f):
     return data_dict
 
 
-def test_residual_vis():
+@corr_shape_parametrization
+def test_corrupt_vis(corr_shape, jones_shape):
+    """
+    Tests corrupt vis against predict_vis. They should do
+    the same thing but corrupt_vis adheres to the structure
+    in the africanus.calibration module.
+    """
+    np.random.seed(42)
```

Comment:
The following is something that I have not done throughout the code base, but I would like to start promoting as a pattern for the future: We should start using numpy random states which parametrise the RNG for the test. So something like:

```python
    rs = np.random.RandomState(42)
    vis = rs.normal((rows, chan, corr)) + rs.normal((rows, chan, corr))*1j
    data_dict = calibration_factory(sigma_n, sigma_f, n_time, ..., rs)
```